docs/docs/fallback-handoff.mdx

```diff
@@ -231,80 +215,147 @@ class ActionDefaultFallback(Action):
+can be overwritten using [Custom Actions](./actions.mdx#custom-actions).
+:::
 
-```bash
-rasa train
-rasa shell
-```
+#### 3. Defining a Two-Stage Fallback rule
 
-Going one step further, if you observe your users asking for certain things, that you'll
-want to turn into a user goal in future, you can handle these as separate intents, to let
-the user know you've understood their message, but don't have a solution quite yet. E.g.,
-let's say the user asks “I want to apply for a job at Rasa”, we can then reply with
-“I understand you're looking for a job, but I'm afraid I can't handle that skill yet.”
+Add the following [Rule](./rules.mdx) to your training data. This rule will make sure
+that the `Two-Stage-Fallback` will be activated whenever a message is received with
+low classification confidence.
 
-```md
-* ask_job
-  utter_job_not_handled
+```yaml
+rules:
+- rule: Implementation of the Two-Stage-Fallback
+  steps:
+  - intent: nlu_fallback
+  - action: two_stage_fallback
+  - form: two_stage_fallback
 ```
 
-:::note
-Here's a minimal checklist of files we modified to help our assistant fail gracefully:
 
-* `data/nlu.yml`:
+## Human Handoff
 
-    * Add training data for the `out_of_scope` intent & any specific out of scope intents that you want to handle seperately
+As part of your fallback action, you may want the bot to hand over to a human agent
+e.g. as the final action in Two-Stage-Fallback, or when the user explicitly asks
+for a human. A straightforward way to achieve human handoff is to configure your
+[messaging or voice channel](messaging-and-voice-channels.mdx) to switch
+which host it listens to based on a specific bot or user message.
 
-* `data/stories.yml`:
+For example, as the final action of Two-Stage-Fallback, the bot could ask the user,
+"Would you like to be transferred to a human assistant?" and if they say yes, the
+bot sends a message with a specific payload like
+e.g. "handoff_to_human" to the channel. When the channel sees this message, it stops listening
+to the Rasa server, and sends a message to the human channel with the transcript
+of the chat conversation up to that point.
 
-    * Add stories for any specific out of scope intents
+The implementation for handing off to a human from the front end will depend on which
+channel you're using. You can
+see an example implementation using an adaption of the [chatroom](https://github.com/scalableminds/chatroom) channel
+in the [Financial Demo](https://github.com/RasaHQ/financial-demo) and
+[Helpdesk-Assistant](https://github.com/RasaHQ/helpdesk-assistant)
+starterpacks.
 
-* `domain.yml`:
 
-    * Add the `out_of_scope` intent & any specific out of scope intents
 
-    * Add an `utter_out_of_scope` response & responses for any specific out of scope intents
+## Summary
 
-* `actions/actions.py`:
+It's up to you how you want your assistant to act in a fallback or handoff
+situation. You might want to use one or more of the options
```

Comment:
maybe we should summarise this with our recommendations, instead of saying "do what you want"?